src/main/resources/io/jenkins/plugins/appcenter/AppCenterRecorder/config.jelly

```diff
@@ -17,7 +17,7 @@
     </f:entry>
 
     <f:entry title="${%Path To App}" field="pathToApp"
-             description="${%Relative path to app.}">
+             description="${%Relative path to app. Placeholders can be used like /your/path/app-*.apk} (the first file found will be used).">
```

Comment:
I think I'd like to keep this as is and put this extra information in the help messages. I realise I haven't got around to writing any of the help messages but I think the description should be kept simple. We can possibly start adding help messages in this PR otherwise there is a separate ticket [[JENKINS-59377]](https://issues.jenkins-ci.org/browse/JENKINS-59377) to address this.

Reply:
I've reverted this one too.